<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSttAddMacroWidget</class>
 <widget class="QDialog" name="QSttAddMacroWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择测试功能</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayoutNameID">
       <item row="0" column="0">
        <widget class="QLabel" name="m_lblItemsName">
         <property name="text">
          <string>名称</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="m_edtItemsID"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="m_edtItemsName"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="m_lblItemsID">
         <property name="text">
          <string>标识</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="m_Btn_OK">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_Btn_Cancel">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
